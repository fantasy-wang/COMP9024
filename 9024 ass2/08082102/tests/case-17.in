badger banger barger darger darter dater mater matey pater patera
