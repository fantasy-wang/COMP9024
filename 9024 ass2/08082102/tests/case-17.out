Dictionary
0: badger
1: banger
2: barger
3: darger
4: darter
5: dater
6: mater
7: matey
8: pater
9: patera
Ordered Word Ladder Graph
V=10, E=11
<0 1> <0 2> 
<1 0> <1 2> 
<2 0> <2 1> <2 3> 
<3 2> <3 4> 
<4 3> <4 5> 
<5 4> <5 6> <5 8> 
<6 5> <6 7> <6 8> 
<7 6> 
<8 5> <8 6> <8 9> 
<9 8> 
Longest ladder length: 9
Longest ladders:
 1: badger -> banger -> barger -> darger -> darter -> dater -> mater -> pater -> patera
